src/types/WalletSetting.ts

```diff
@@ -2,6 +2,18 @@ import { IProviderDisplay, ThemeColors } from 'web3modal'
 // eslint-disable-next-line import/no-unresolved
 import { IOptions } from 'web3modal/dist/providers/connectors/torus'
 
+type WalletConnectConnectorOptions = {
```

Comment:
We should use it
https://github.com/Web3Modal/web3modal/blob/master/src/providers/connectors/walletconnect.ts#L3

Reply:
@tomonari-t -san. I have updated it.

Reply:
@longlh1997 Thanks!